docs/code.rst

```diff
@@ -120,27 +147,52 @@ Community-contributed Operators
 .. autoclass:: airflow.contrib.operators.file_to_gcs.FileToGoogleCloudStorageOperator
 .. autoclass:: airflow.contrib.operators.file_to_wasb.FileToWasbOperator
 .. autoclass:: airflow.contrib.operators.gcs_copy_operator.GoogleCloudStorageCopyOperator
 .. autoclass:: airflow.contrib.operators.gcs_download_operator.GoogleCloudStorageDownloadOperator
 .. autoclass:: airflow.contrib.operators.gcs_list_operator.GoogleCloudStorageListOperator
 .. autoclass:: airflow.contrib.operators.gcs_operator.GoogleCloudStorageCreateBucketOperator
+.. autoclass:: airflow.contrib.operators.gcs_to_bq.GoogleCloudStorageToBigQueryOperator
 .. autoclass:: airflow.contrib.operators.gcs_to_gcs.GoogleCloudStorageToGoogleCloudStorageOperator
+.. autoclass:: airflow.contrib.operators.hipchat_operator.HipChatAPIOperator
+.. autoclass:: airflow.contrib.operators.hipchat_operator.HipChatAPISendRoomNotificationOperator
+.. autoclass:: airflow.contrib.operators.jira_operator.JiraOperator
+.. autoclass:: airflow.contrib.operators.kubernetes_pod_operator.KubernetesPodOperator
+.. autoclass:: airflow.contrib.operators.mlengine_operator.MLEngineBatchPredictionOperator
+.. autoclass:: airflow.contrib.operators.mlengine_operator.MLEngineModelOperator
+.. autoclass:: airflow.contrib.operators.mlengine_operator.MLEngineVersionOperator
+.. autoclass:: airflow.contrib.operators.mlengine_operator.MLEngineTrainingOperator
+.. autoclass:: airflow.contrib.operators.mysql_to_gcs.MySqlToGoogleCloudStorageOperator
+.. autoclass:: airflow.contrib.operators.postgres_to_gcs_operator.PostgresToGoogleCloudStorageOperator
 .. autoclass:: airflow.contrib.operators.pubsub_operator.PubSubTopicCreateOperator
 .. autoclass:: airflow.contrib.operators.pubsub_operator.PubSubTopicDeleteOperator
 .. autoclass:: airflow.contrib.operators.pubsub_operator.PubSubSubscriptionCreateOperator
 .. autoclass:: airflow.contrib.operators.pubsub_operator.PubSubSubscriptionDeleteOperator
 .. autoclass:: airflow.contrib.operators.pubsub_operator.PubSubPublishOperator
-.. autoclass:: airflow.contrib.operators.hipchat_operator.HipChatAPIOperator
-.. autoclass:: airflow.contrib.operators.hipchat_operator.HipChatAPISendRoomNotificationOperator
 .. autoclass:: airflow.contrib.operators.qubole_operator.QuboleOperator
+.. autoclass:: airflow.contrib.operators.sftp_operator.SFTPOperator
+.. autoclass:: airflow.contrib.operators.spark_jdbc_operator.SparkJDBCOperator
+.. autoclass:: airflow.contrib.operators.spark_sql_operator.SparkSqlOperator
+.. autoclass:: airflow.contrib.operators.spark_submit_operator.SparkSubmitOperator
+.. autoclass:: airflow.contrib.operators.sqoop_operator.SqoopOperator
 .. autoclass:: airflow.contrib.operators.ssh_operator.SSHOperator
 .. autoclass:: airflow.contrib.operators.vertica_operator.VerticaOperator
-.. autoclass:: airflow.contrib.operators.vertica_to_hive.VerticaToHiveTransfer
 .. autoclass:: airflow.contrib.operators.spark_submit_operator.SparkSubmitOperator
 
+Transfers
```

Comment:
same here, can be combined with operators.

Another reason is that there are operators above that are technically transfers but not named Transfer (such as `MySqlToGoogleCloudStorageOperator ` or `PostgresToGoogleCloudStorageOperator`).

Reply:
Yeah, I wondered about those… 

I think it's somewhat useful to have all the transfers separated out, but if they don't actually all implement a common interface it probably does make sense to merge them back in to the main list above. 

Another thing I considered was trying to split out all of the flow control operators (`BranchPython`, `ShortCircuit`, `TriggerDagRun`, etc) into their own section, but I haven't really worked with any of these enough to make that call :)